server/src/aws/index.ts

```diff
@@ -0,0 +1,10 @@
+import AWS from 'aws-sdk';
+import { aws } from '../config';
+
+//configuring the AWS environment
```

Comment:
nitpick - add a space in comment 😄 
```suggestion
// configuring the AWS environment
```

Reply:
Lemme do this in another PR that enforces documentation